### PR TITLE
Test against developer version of Sphinx

### PR DESCRIPTION
**Pull request by astrofrog**
cc @bsipocz

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +30,7 @@ env:
     - PYTHON_VERSION=2.7
     - PYTHON_VERSION=3.4 SPHINX_VERSION=1.3
     - PYTHON_VERSION=3.5
+    - PYTHON_VERSION=3.6 SPHINX_VERSION=dev CONDA_DEPENDENCIES="setuptools pytest cython pytest-cov"
```
Comment (bsipocz) [inline .travis.yml]: pytest is always installed with ``ci-helpers``, you don't need to list it

**Comment by bsipocz** [comment]
Looks good to me. Could you also enable cron for travis?

**Comment by astrofrog** [comment]
I've added a weekly cron job

**Comment by astrofrog** [comment]
Note that the remaining failure was due to the change in API in https://github.com/sphinx-doc/sphinx/pull/3668

**Pull request merged**